practices/graded-browser-support.md

```diff
@@ -13,47 +13,60 @@ This page describes the way that we support different browsers and browser versi
+
+## Rationale
+
+We have three reasons for this:
+
+* We wish to support all users. No matter their device, browser, or network condition, the user should find a working and robust product.
+* We wish to keep support costs low. Given the previous statement, we do not want to spend time [making advanced CSS and JavaScript work in older browsers](https://en.wikipedia.org/wiki/Pareto_principle#In_software).
 
 ## Graded browser support list
 
 This is the current list of browser versions and their corresponding grades:
 
-| Tables          | Grade-A support                 | Grade-C support    |
+| Browser         | Advanced                        | Core               |
 | --------------- |:-------------------------------:| ------------------:|
 | Chrome          | latest stable, latest stable -1 | < 29               |
-| Edge            | latest stable, latest stable -1 | none               |
+| Edge            | latest stable, latest stable -1 | n/a                |
 | Firefox         | latest stable, latest stable -1 | < 29               |
 | IE              | 11                              | < 11               |
 | Opera           | latest stable                   | < 16               |
 | Safari iOS      | latest stable, latest stable -1 | < 7                |
 | Safari MacOS    | latest stable, latest stable -1 | < 6.1              |
 | Android Webview | latest stable                   | < 30 (Android 4.4) |
 
-Every unlisted browser version is Grade-X.
+### Grey area browsers
+
+Some browser versions exist in the grey area outside and/or between Advanced and Core grades. For example:
+
+* Old versions of an evergreen browser: a user has turned off auto-updates, or their upgrade opportunities are limited by their device or administrator.
+* Nightly or developer versions of evergreen browsers.
+
+We serve these grey area browsers the Advanced version of a site, so they will receive the full experience. Due to resourcing implications we do not specifically test our products against them - however we do expect them to work. Over time, analysis of errors and usage patterns may cause some browsers in this grey area to be deliberately changed to either Advanced or Core, to allow for better support.
 
 ## Implementing graded browser support
 
-Our primary approach to Graded Browser Support is the "[Cutting the Mustard (CTM)](http://responsivenews.co.uk/post/18948466399/cutting-the-mustard)" progressive enhancement technique, based upon the principles developed by the BBC.
+Our primary approach to implementing Graded Browser Support is the "[Cutting the Mustard (CTM)](http://responsivenews.co.uk/post/18948466399/cutting-the-mustard)" progressive enhancement technique, based upon the principles developed by the BBC.
```

Comment:
Not sure if we need to change the name on that basis alone. We still call Cutting the Mustard "Cutting the Mustard" despite it not being the same as the original BBC technique. 

However, maybe as the scale it isn't _graded_ anymore - we're describing the results as something more boolean rather than many possible values - it might make sense to change the name for that reason?

Reply:
I was under the impression CTM is still the same, it's just a different media query. That will happen naturally over time.

I'm not sure having two "grades" makes our approach to GBS still _GBS_ as it does away with Grade-X, and the buckets no longer have the word "Graded" in the name. Having two buckets doesn't really implies "grades" to me, it's more of a dialectic, as you say. That's certainly debatable tough! 

OTOH changing the name now means more re-education though when really not much has changed 🤔 

I think we should at least call out "our version" or "our application" of GBS though, as to the casual googler finding this resource and some other resource about GBS it will be confusing.

Reply:
@jpw CTM isn't the same - The BBC approach used JavaScript feature detection, not CSS media queries.

Reply:
> I think we should at least call out "our version" or "our application" of GBS though, as to the casual googler finding this resource and some other resource about GBS it will be confusing.

👍 to that. I don't think changing the name is necessary at this point.

Reply:
I would suggest that what we do doesn't even need a name.

The document could be called just "Browser support", which is what I would be looking for if I wasn't familiar with our work, btw. Calling the document "Graded browser support" could cause someone not familiar with it to interpret as if we have more than one kind of browser support.

In my opinion we could just call the doc "Browser support" and then in the text just explain "the was that we support browsers is a variation of Yahoo's Graded Browser etc etc etc".

This is also relevant because iirc we have a document on Hive which is basically the same as this one. I would think that people would search more for "browser support" than anything else.

Reply:
@josebolos do like that idea

Reply:
@josebolos genius. I like it.